### ISSUE TITLE
Hidden suit - variant

### ISSUE THREAD

**Issue by Romain672** [Zamiell/hanabi-live#705]
Each card of this suit will just ignore the first clue given to him.

So, in hand : b1-b3-h1-h4-r5, if you clue 1, only the 1 on the b1 will be visible (the h1 will ignore the clue).
Then, if on this same hand, you clue the hidden color, it will color only the h1 since it was clued earlier, but will ignore the h4.

Again, on this same hand, if in the first turn, you clue the hidden color instead of 1, nothing will happen. But the player will know he have at least one hidden card (if empty clue are off).

**Comment by Zamiell** [comment]
is the hidden suit touched by all clues?

**Comment by Romain672** [comment]
No, I think it's better if you force the person to try to clue this suit a lot.

**Comment by Zamiell** [comment]
so the suit has its own color?

**Comment by Romain672** [comment]
yes

**Comment by Zamiell** [comment]
are empty clues allowed for the first touch or not?

**Comment by Romain672** [comment]
Yes, starting the game with hidden color would just show the player he have, at least, 1 hidden card (but whatever choice is fine for me).
